<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceConnectedView</class>
 <widget class="QWidget" name="SurfaceConnectedView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>317</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Surface</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="SurfaceSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="ApplyBtn">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SaveBtn">
      <property name="text">
       <string>Save Reslut</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ImportBtn">
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Plot" name="ColorEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>Plot</class>
   <extends>QFrame</extends>
   <header>plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
